packages/slice-machine/src/components/PageLayout/PageLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import type { FC, ReactNode } from "react";
+import { Text } from "@prismicio/editor-ui";
+
+import * as styles from "./PageLayout.css";
+
+type PageLayoutProps = Readonly<{
+  children?: ReactNode;
+}>;
+
+export const PageLayout: FC<PageLayoutProps> = ({
```

Comment:
❓ To create component you prefer this way with FC? 
To have a coherent way to do in the future. Personally I don't really care, function, arrow function, arrow function with FC 😄

Reply:
No problem honestly on my side, again I will do the same to try not having 1000 ways to do :)